bigbluebutton-html5/imports/ui/components/settings/component.jsx

```diff
@@ -218,6 +218,9 @@ class Settings extends Component {
         confirm={{
           callback: () => {
             this.updateSettings(this.state.current);
+            if (window.location.pathname.includes('users')) {
```

Comment:
Would be better to save the current path and push it after the settings update. What happend if you update the settings with a chat open? (: